### PR TITLE
someone forgot to add the helltaker skirt to the loadout

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

title

## How This Contributes To The Skyrat Roleplay Experience

stuff should exist

## Changelog

:cl:
expansion: ACTUALLY added the helltak- inferno skirt to the loadout
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm [hunk 1]
```diff
@@ -913,10 +913,15 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	name = "Basil Uniform"
 	item_path = /obj/item/clothing/under/costume/basil_boys
 
+//ht
 /datum/loadout_item/under/formal/inferno
 	name = "Inferno Suit"
 	item_path = /obj/item/clothing/under/rank/civilian/lawyer/inferno
 
+/datum/loadout_item/under/formal/inferno_skirt
+	name = "Inferno Suit"
```
Comment (thestubborn) [inline modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]:
```suggestion
	name = "Inferno Skirt"
```

File: modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm [hunk 2]
```diff
@@ -913,10 +913,15 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	name = "Basil Uniform"
 	item_path = /obj/item/clothing/under/costume/basil_boys
 
+//ht
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]: This doesn't really provide any further insight, it's kinda just there to be there and those comments are pretty useless.
```suggestion
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm (hunk 2 above)
Comment (thestubborn) [reply modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]: eh fair

**Comment by thestubborn** [comment]
@GoldenAlpharex @Gandalf2k15

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**